apidoc/Titanium/UI/OptionBar.yml

```diff
@@ -60,6 +60,30 @@ properties:
     default: horizontal
     availability: creation
 
+  - name: selectedBackgroundColor
```

Comment:
I feel like these properties (partially) are available on iOS as well, but under a different naming

Reply:
EDIT: Yep, there is:
- tintColor (for the selected background item)
- backgroundColor (for the global background of the option bar, could be seen as unselected background color)
- textColor (unselected text color)
- selectedTextColor (...)
- selectedButtonColor (same as tintColor, but an older API)

Reply:
Are all theses available in an OptionBar? I only see `selectedButtonColor` in a ButtonBar. 
I'll move from `selectedBackgroundColor` -> `selectedButtonColor` and `textColor` -> `color`. The rest looks the same.

Reply:
They are technically exposed, but maybe not documented? If it's not intended to be documented for this component (they both use the UISegmentedControl internally), we can keep this one as-is.